homework_19_070223/src/People.java

```diff
@@ -0,0 +1,3 @@
+public interface People {
```

Comment:
Not correct interface naming

Reply:
Fixed. Renamed to interface Checkable {}